pkg/ctl/scale/nodegroup.go

```diff
@@ -31,6 +31,8 @@ func scaleNodeGroupWithRunFunc(cmd *cmdutils.Cmd, runFunc func(cmd *cmdutils.Cmd
 	cmd.FlagSetGroup.InFlagSet("General", func(fs *pflag.FlagSet) {
 		fs.StringVar(&cfg.Metadata.Name, "cluster", "", "EKS cluster name")
 		fs.StringVarP(&ng.Name, "name", "n", "", "Name of the nodegroup to scale")
+		cmdutils.AddConfigFileFlag(fs, &cmd.ClusterConfigFile)
+		cmdutils.AddNodeGroupFilterFlags(fs, &cmd.Include, &cmd.Exclude)
```

Comment:
If a command is only going to be applied to one nodegroup and doesn't support using globs I think it would be a better UX to use `--name` instead of `--include` and `--exclude`. It is also less complex code-wise.

@mb @cPu1 What do you think?

Reply:
Please let me know if the above proposal is confirmed, thanks :)

Reply:
Confirmed as far as I'm concerned, I think @cPu1 too since he also :+1:ed it

Reply:
cool, I can continue with this PR soon, hope that `@mb` will give a go ahead as well :pray:

Reply:
@sayboras I think I'm mb :grin:

Reply:
didn't have time to wrap it up this week, will plan sometime this weekend or next week if you don't mind

Reply:
Sure! No hurries!